CAP/gap/ConstructiveCategoriesRecord.gi

```diff
@@ -73,7 +73,15 @@ IsSymmetricClosedMonoidalCategory  := Concatenation( [
 IsRigidSymmetricClosedMonoidalCategory  := Concatenation( [ 
 "TensorProductInternalHomCompatibilityMorphismInverseWithGivenObjects",
 "MorphismFromBidualWithGivenBidual"
-], ~.IsSymmetricClosedMonoidalCategory )
+], ~.IsSymmetricClosedMonoidalCategory ),
+
+IsAbelianCategoryWithEnoughProjectives := Concatenation( [
+"EpimorphismFromSomeProjectiveObject"
+], ~.IsAbelianCategory ),
```

Comment:
What about the lift functions?

Reply:
You are right, I will update the pull request.